lib/cloud_controller/dea/app_stager_task.rb

```diff
@@ -80,8 +103,8 @@ def handle_first_response(response, error, promise)
 
       def handle_second_response(response, error)
         @multi_message_bus_request.ignore_subsequent_responses
-        ensure_staging_is_current!
-        check_staging_error!(response, error)
+        ensure_staging_is_current!(false)
```

Comment:
Discussed in Slack, @fraenkel and I are on the same page regarding checking of task_id now.  Only change required is to move @app.refresh earlier in the NATS path.